src/binder/bind/bind_graph_pattern.cpp

```diff
@@ -27,9 +27,34 @@ Binder::bindGraphPattern(const std::vector<std::unique_ptr<PatternElement>>& gra
     return std::make_pair(std::move(queryGraphCollection), std::move(propertyCollection));
 }
 
+void static addUndirectedTableNames(const PatternElement& patternElement) {
```

Comment:
rename to `rewriteNodeTableNameForUndirectedRel` 

also put the comment above the function signature. Change the comment to


"For undirected pattern (A)-[R]-(B), we need to match R in both FWD and BWD direction. Since computation always start from one node, we need to rewrite node table name to be the union of both node table names, i.e. (A|B)-[R]-(A|B)"